cmasher/tests/test_utils.py

```diff
@@ -65,6 +66,122 @@ def _MPL38_colormap_eq(cmap, other) -> bool:
 
 
 # %% PYTEST CLASSES AND FUNCTIONS
+# Pytest class for combine_cmaps
+class Test_combine_cmaps:
+    # Test if multiple Colormaps or colormap names can be combined
+    @pytest.mark.parametrize(
+        "cmaps, nodes",
+        [
+            (["Blues", "Oranges", "Greens"], [0.25, 0.75]),
+            (["Blues", "Oranges", "Greens"], np.array([0.25, 0.75])),
+            (
+                [
+                    mpl.colormaps["Blues"],
+                    mpl.colormaps["Oranges"],
+                    mpl.colormaps["Greens"],
+                ],
+                [0.25, 0.75],
+            ),
+        ],
+    )
+    def test_combine_cmaps(self, cmaps, nodes):
+        combined_cmap = combine_cmaps(cmaps, nodes=nodes, n_rgb_levels=256)
+        blues_cmap = mpl.colormaps["Blues"]
+        oranges_cmap = mpl.colormaps["Oranges"]
+        greens_cmap = mpl.colormaps["Greens"]
+
+        assert np.allclose(combined_cmap(0.0), blues_cmap(0))
+        assert np.allclose(combined_cmap(0.25), oranges_cmap(0))
+        assert np.allclose(combined_cmap(0.75), greens_cmap(0))
+        assert np.allclose(combined_cmap(1.0), greens_cmap(255))
+
+        assert combined_cmap.N == 256
+
+    # Test if combining less than 2 colormaps triggers an error
+    @pytest.mark.parametrize(
+        "cmaps",
+        [
+            pytest.param([], id="no_cmap"),
+            pytest.param(
+                [
+                    "Blues",
+                ],
+                id="single_cmap",
```

Comment:
a note on automated formatting here: these would normally fit on a single line and there's no reason to expand over multiple lines, but that's what `black` does if you add trailing comas after the last item in a list or a set of functions arguments. Let's remove those and let `black` re-format to a more-readable one-liner
```suggestion
                [
                    "Blues"
                ],
                id="single_cmap"
```

Reply:
Thanks a lot! That was confusing me 😃.